### PR TITLE
chore: add a troubleshooting section to the Debian packages upgrade page

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: None
Ticket: None

### CODE REVIEW

File: 03.Client-installation/02.Install-with-Debian-package/01.Upgrading/docs.md [hunk 1]
```diff
@@ -87,6 +87,23 @@ In addition, this package will also automatically remove:
 After the upgrade, please update your scripts invoking the `mender` CLI interface to use the
 correct binaries (`mender-auth` or `mender-update`).
 
+If the Mender add-ons Debian packages (e.g., `mender-connect` and `mender-configure`) are already
+installed in the system, the upgrade can fail with an error message similar to this:
```
Comment (lluiscampos) [inline 03.Client-installation/02.Install-with-Debian-package/01.Upgrading/docs.md:91]: It only affects old `dpkg`/`apt`. I would rephrase like:
```suggestion
For old Debian and Ubuntu distributions, if the Mender add-ons packages (e.g., `mender-connect` and `mender-configure`) are already
installed in the system, the upgrade can fail with an error message similar to this:
```

File: 03.Client-installation/02.Install-with-Debian-package/01.Upgrading/docs.md [hunk 2]
```diff
@@ -87,6 +87,23 @@ In addition, this package will also automatically remove:
...
+ mender-client4 : Conflicts: mender-client but 1:3.5.2-1+debian+buster is to be installed
+E: Unable to correct problems, you have held broken packages.
+```
+
+In this case, ensure to upgrade the Mender add-ons Debian packages at the same time, for
+example running:
+
+<!--AUTOMATION: ignore -->
+```bash
+apt-get install mender-client4 mender-connect mender-configure
```
Comment (lluiscampos) [inline 03.Client-installation/02.Install-with-Debian-package/01.Upgrading/docs.md:105]: Have you tested this explicitly? I have only tested the workaround documented in the [test](https://github.com/mendersoftware/mender-dist-packages/blob/ab857f522e48888998f9ae38cdb6bb1ca07cc51a/tests/test_install_mender_sh.py#L273):
```suggestion
apt-get install mender-update mender-client4
```

Comment (tranchitella) [reply 03.Client-installation/02.Install-with-Debian-package/01.Upgrading/docs.md:105]: Yes, though your command is better so I'm applying it.

**Review by lluiscampos — APPROVED** [review]
Thank you
